changelog.md

```diff
@@ -1,10 +1,21 @@
 
 
+## 3.3.1 / 2020-12-17
+
+### Fixed Issues
+
+- Note: [#3722](https://github.com/tracim/tracim/issues/3722)
+- Mention: [#3927](https://github.com/tracim/tracim/issues/3927)
+- Frontend: [#3542](https://github.com/tracim/tracim/issues/3542),
+[#3809](https://github.com/tracim/tracim/issues/)
+
+
 ## 3.3.0 / 2020-12-02
 
 ### New Features
 
 - Personal and per-space activity feeds help you keep up with what is going on in Tracim: you will be able to notice new contents, new members and other updates in real time at a glance.
+- Access to applications through tabs (dashboard, contents, activity feed, etc)
```

Comment:
```suggestion
- Tabs to access the dashboard contents and activity feed
```

(There is nothing else for now, I think it's clearer like this)

Reply:
@raphj I'm not sure about this. Tabs not just shown dashboard content and activity feed, so I think its better to find wording to explain the futur tabs can be added in this view.